core/src/components/input/input.tsx

```diff
@@ -541,6 +558,10 @@ export class Input implements ComponentInterface {
     this.emitInputChange(ev);
   };
 
+  private showPassword = () => {
```

Comment:
```suggestion
  private togglePasswordVisibility = () => {
```

Nit: This name isn't entirely accurate because depending on `this.type` it will either show or hide the password.

Reply:
True!